sys-kernel/buildkernel/buildkernel-1.0.37.ebuild

```diff
@@ -8,7 +8,7 @@ inherit eutils
 DESCRIPTION="Build secure boot EFI kernel with LUKS, LVM and plymouth"
 BASE_SERVER_URI="https://github.com/GenPi64"
 HOMEPAGE="${BASE_SERVER_URI}/${PN}"
-SRC_URI="${BASE_SERVER_URI}/${PN}/releases/download/${PV}/${P}.tar.gz"
+SRC_URI="${BASE_SERVER_URI}/${PN}/archive/refs/tags/${PV}.tar.gz"
```

Comment:
I think this would need `-> ${PN}` so that the downloaded file would get renamed to `buildkernel-1.0.37.tar.gz` to prevent collisions with other packages.